executor/builder.go

```diff
@@ -1245,7 +1245,7 @@ func (b *executorBuilder) buildMemTable(v *plannercore.PhysicalMemTable) Executo
 	e := &TableScanExec{
 		baseExecutor:   newBaseExecutor(b.ctx, v.Schema(), v.ExplainID()),
 		t:              tb,
-		columns:        v.Table.Columns,
+		columns:        v.Columns,
```

Comment:
Why need this change?

Reply:
For cluster memory table reader, will have prune column, so the `TableScanExec` should only fetch the related column, not all columns of the table.

Reply:
Add some comments for the field.